### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by fstirlitz**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [x] Improvement

---

A couple of unrelated small fixes that I didn’t particularly feel like submitting separately and I hope will be uncontroversial. Commit messages should be self-explanatory.

### CODE REVIEW

File: yt_dlp/extractor/common.py [hunk 1]
```diff
@@ -2876,7 +2876,10 @@ def _parse_ism_formats_and_subtitles(self, ism_doc, ism_url, ism_id=None):
             stream_name = stream.get('Name')
             stream_language = stream.get('Language', 'und')
             for track in stream.findall('QualityLevel'):
-                fourcc = track.get('FourCC', 'AACL' if track.get('AudioTag') == '255' else None)
+                fourcc = track.get('FourCC', '')
+                if not fourcc:
+                    if track.get('AudioTag') == '255':
+                        fourcc = 'AACL'
```
Comment (pukkandan) [inline yt_dlp/extractor/common.py:2882]:
```py
fourcc = track.get('FourCC') or ('AACL' if track.get('AudioTag') == '255' else None)
```
is simpler, no?

**Comment by pukkandan** [comment]
btw, if you find small issues like this in the future that you don't feel like making PRs for, you can either just open an issue or just ping me on discord (since github doesnt have DMs...)

File: yt_dlp/extractor/common.py (hunk 1 above)
Comment (fstirlitz) [reply yt_dlp/extractor/common.py:2882]: Yes, I guess so.

**Comment by fstirlitz** [comment]
I don’t use Discord. And since I have already written the patches, I might as well submit them this way.

I also added a patch removing unsupported Python versions from CI.

If you’d prefer not to merge any of those just yet, you can just cherry-pick the ones you want locally.

**Comment by pukkandan** [comment]
>  I have already written the patches, I might as well submit them this way.

Sure, this approach is fine as well. I was just thinking an issue/DM would be lower effort

**Comment by pukkandan** [comment]
* I was also working on cleaning up the tests, so I pushed my version of it and not yours
* I didn't delete `swfinterp`, but marked it as unused since this is what I had done with some other unused files of youtube-dl

The other patches have been committed

**Pull request closed without merging**